subsys/bluetooth/controller/ll_sw/nordic/lll/lll_sync_iso.c

```diff
@@ -776,6 +773,39 @@ static void isr_rx(void *param)
 #endif /* HAL_RADIO_GPIO_HAVE_LNA_PIN */
 }
 
+static void isr_rx_iso_data_valid(struct lll_sync_iso *lll, uint8_t bis_idx,
+				  struct node_rx_pdu *node_rx)
+{
+	struct node_rx_iso_meta *iso_meta;
+
+	node_rx->hdr.type = NODE_RX_TYPE_ISO_PDU;
+	node_rx->hdr.handle = lll->stream_handle[bis_idx];
+
+	iso_meta = &node_rx->hdr.rx_iso_meta;
+	iso_meta->payload_number = lll->payload_count + (lll->bn_curr - 1U) +
+				   (lll->ptc_curr * lll->pto) - lll->bn;
+	iso_meta->timestamp = HAL_TICKER_TICKS_TO_US(radio_tmr_start_get()) +
+			      radio_tmr_aa_restore() - addr_us_get(lll->phy) +
+			      (ceiling_fraction(lll->ptc_curr, lll->bn) *
+			       lll->iso_interval * PERIODIC_INT_UNIT_US);
```

Comment:
Are these calculations from the core spec? If so, please add a reference to the section, and if not please ignore this comment :)

Reply:
This is not from spec, but vendor specific capture of the anchor point for a ISO interval.